bindings/rust/build.rs

```diff
@@ -235,6 +235,9 @@ fn main() {
     }
     if target_os.eq("zkvm") {
         cc.define("__ZKVM__", None);
+        //if env::var("RISC0_UNCHECKED").is_ok() {
+        //    cc.define("__RISC0_UNCHECKED__", None);
+        //}
```

Comment:
Maybe add a comment explaining what this does. Perhaps something like this
```suggestion
        // // Allows the use of precompiles that are faster but which can be misused insecurely
        //if env::var("RISC0_UNCHECKED").is_ok() {
        //    cc.define("__RISC0_UNCHECKED__", None);
        //}
```

Reply:
Ok, I added this comment now.